README.md

```diff
@@ -21,6 +21,31 @@ Set these environment variables if you need to change their defaults
 | TEMPORAL_HOT_RELOAD_TEST_PORT | HTTP port used by hot reloading in tests                          | 8082           |
 | TEMPORAL_EXTERNAL_SCRIPTS     | Addtional JavaScript tags to serve in the UI                      |                |
 
+### Configuring Authentication (optional)
+
+Since v1.2, Temporal Web offers optional Oauth SSO authentication. You can enable it by adding a `server/config.yml` file:
```

Comment:
i would defer to @feedmeapples to answer about the https but in future i would definitely like to enable auth through the UI instead of just changing the file.

also the wording is a little off:

```suggestion
Since v1.2, Temporal Web offers optional Oauth SSO authentication. You can enable it by changing the `server/config.yml` file:
```